Demo/src/main/java/com/paypal/android/ui/features/FeaturesView.kt

```diff
@@ -98,14 +115,20 @@ fun FeatureOptions(
 
 @Composable
 fun FeatureGroupHeader(text: String) {
-    Spacer(modifier = Modifier.size(UIConstants.paddingLarge))
-    Text(
-        text = text,
-        color = MaterialTheme.colorScheme.onSurface,
-        fontWeight = FontWeight.Bold,
-        style = MaterialTheme.typography.titleLarge,
-    )
-    Spacer(modifier = Modifier.size(UIConstants.paddingMedium))
+    Row(
```

Comment:
Moving away from `Spacer` views here. Padding does the same job without adding unnecessary Compose elements.